prov/tcp/src/tcpx_progress.c

```diff
@@ -42,8 +42,32 @@
 #include <ofi_util.h>
 #include <ofi_iov.h>
 
+void auto_prog_mgr_close(struct auto_prog_mgr *mgr)
```

Comment:
Please use prefix tcpx_ in all functions for this provider to avoid name space collisions and allow easy identification of code ownership.